addons/website_sale_collect/static/src/js/click_and_collect_availability/click_and_collect_availability.js

```diff
@@ -11,6 +11,7 @@ export class ClickAndCollectAvailability extends Component {
     static template = 'website_sale_collect.ClickAndCollectAvailability';
     static props = {
         productId: Number,
+        active: {type: Boolean, default: true},
```

Comment:
That's not the way to declare default props. See https://github.com/odoo/owl/blob/master/doc/reference/props.md#default-props
```suggestion
        active: {type: Boolean, optional: true},
```
```js 
static defaultProps = {
    active: true,
}
```